scripts/modules/MsIndustryLinks/publish/application/New-AzureApplicationOffer.md

```diff
@@ -0,0 +1,81 @@
+# New-AzureApplicationOffer
+
+Module: [MsIndustryLinks](../../README.md)
+
+Creates an Azure Application offer for an Industry Link.
+
+## Syntax
+
+```powershell
+New-AzureApplicationOffer
+    -AssetsDirectory <String>
+    -MarketplacePackageZipFile <String>
+    -OfferId <String>
+    -PlanId <String>
+```
+
+## Description
+
+Creates an Azure Application (solution template) offer in Partner Center for an Industry Link that is ready for publishing. This offer will enable your customers to install your Industry Link from the Azure Marketplace. For more information, see [Plan a solution template for an Azure application offer](https://learn.microsoft.com/en-us/partner-center/marketplace/plan-azure-app-solution-template).
+
+## Examples
+
+### Example 1: Create an Azure Application offer for the Industry Link
+
+```powershell
+New-AzureApplicationOffer
+    -AssetsDirectory listingAssets
+    -MarketplacePackageZipFile marketplacePackage.zip
+    -OfferId contoso-industry-link
+    -PlanId contoso-industry-link
+```
+
+## Parameters
+
+### -AssetsDirectory
+
+The path to the offer assets folder containing the offer logos, listing file, and manifest.yml file. See [assets](assets) for examples.
+
+|                |                                                                                                                       |
+| -------------- | --------------------------------------------------------------------------------------------------------------------- |
+| Type:          | [String](https://learn.microsoft.com/en-us/powershell/scripting/lang-spec/chapter-04?view=powershell-7.3#431-strings) |
+| Default value: | None                                                                                                                  |
+
+### -MarketplacePackageZipFile
+
+The path to the marketplace package zip file (marketplacePackage.zip) that is generated by the [New-AzureDeploymentPackage](../../azureDeploymentPackage/New-AzureDeploymentPackage.md) cmdlet. This file should be named "marketplacePackage.zip".
```

Comment:
Is there a hard requirement that the file needs to be named "marketplacePackage.zip"? If so, why do we need to pass it in as a parameter?

Reply:
Yeah, for now, it's what the script is looking for. It still needs the location of the file.